app/models/message.rb

```diff
@@ -107,4 +109,11 @@ def process_written_reasons
   def claim_updater
     Claims::ExternalUserClaimUpdater.new(claim, current_user: sender)
   end
+
+  def duplicate_message_attachment
+    return unless attachment.attached?
+
+    attachment_blob = attachment.blob
+    attachments.attach(attachment_blob) unless attachments.find_by(blob: attachment_blob)
```

Comment:
I'm not entirely sure (need to check) but I think it is not possible to attach the same blob twice, in which case this can be reduced to:

```suggestion
    attachments.attach(attachment.blob)
```

However, this is only temporary so it isn't important if it turns out to be more complicated.

Reply:
Yes, I just checked you are right.